core/src/err/mod.rs

```diff
@@ -779,6 +779,142 @@ pub enum Error {
 	/// The db is running without an available storage engine
 	#[error("The db is running without an available storage engine")]
 	MissingStorageEngine,
+
+	/// The requested analyzer already exists
+	#[error("The analyzer '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	AzAlreadyExists {
```

Comment:
How about renaming this to `AnalyzerAlreadyExists`? This is part of the public API. Currently one has to read the documentation to decipher what `Az` means.

Reply:
Considering all related errors have the same naming scheme, I prefer the shorthand name